### DISCUSSION TITLE
LibreScore is an alternative to MuseScore in what sense?

### DISCUSSION THREAD

**Discussion by HubKing** [LibreScore/app-librescore#47] [Q&A] [answered]
When I first read the description on the LibreScore website, I thought it was an independent service where users share sheet music. But the LibreScore website seems to have no feature to browser/search sheet music. So, I downloaded this app, thinking that maybe the service is only accessible through this app. 

But when I ran the app, it seems that this is just a client for MuseScore, and none of the downloading (MCSZ, MIDI, PDF) worked; it showed "Server error". I guess that is because, of course, MuseScore's server blocked this client, but what about the LibreScore being an alternative? Doesn't LibreScore have its own repository for sheet music?

**Comment by PeterNjeim** [comment, marked as answer]
Edit: similar discussion was here: https://github.com/LibreScore/app-librescore/discussions/25

The defunct LibreScore website (https://github.com/LibreScore/LibreScore) has received no updates since March of 2021. It was a service where users could upload their sheets to IPFS, though there was no mechanism to retrieve others' sheets, only your own. It is illegal to distribute copyrighted content, and so the project was stopped.

The app, like all the programs in the LibreScore suite, has no relation to the website. The app is one of the 4 installable programs of the LibreScore suite. It lets you download MSCZ, MIDI, and MP3 files, as described in the table of the main LibreScore GitHub repository (https://github.com/LibreScore/dl-librescore), whose description is simply "Download sheet music". As expected, all the programs of the LibreScore suite let you download sheet music.

As for your "server error", there's an ID code that is shown as well, and it's used to debug your issue on our server-side. Please do not share it here, all support-related requests are conducted on the Discord server. MuseScore did not block anything, the issue unfortunately is either on your end or our end.

The repository of sheet music (https://github.com/Xmader/musescore-dataset) has been discontinued since it's illegal. We have no plans to distribute copyrighted content. If you're not understanding, here's a simple history:

Xmader started this project with the intention of making every song on MuseScore free. He was successful and people used his viral repository (https://github.com/Xmader/musescore-downloader). I discovered LibreScore in late 2020 and notified Xmader that the dataset was illegal. After lots of debate, I was accepted as a contributor and started working on a legal method of downloading MSCZ files, culminating in the creation of the LibreScore App. All the information related to the dataset and distributing sheet music has been outdated for a few years now, and LibreScore is now simply a suite of downloading and conversion tools.
